ExportQueryAPI.go

```diff
@@ -0,0 +1,195 @@
+package main
+
+import (
+	"bytes"
+	"database/sql"
+	"encoding/json"
+	"fmt"
+	"io"
+	"log"
+	"net/http"
+	"os"
+	"time"
+
+	"github.com/aws/aws-lambda-go/events"
+	"github.com/aws/aws-lambda-go/lambda"
+	"github.com/aws/aws-sdk-go/aws"
+	"github.com/aws/aws-sdk-go/aws/session"
+	"github.com/aws/aws-sdk-go/service/emrserverless"
+	_ "github.com/lib/pq"
+)
+
+type RequestBody struct {
+	Query       string `json:"query"`
+	Destination string `json:"destination"`
+}
+
+type SuccessResponse struct {
+	JobId     string `json:"jobId"`
+	JobStatus string `json:"jobStatus"`
+}
+
+type FailureResponse struct {
+	Message string `json:message`
+}
+
+func Validate(token string, query string, vaultId string) string {
+	if len(query) == 0 {
+		log.Printf("Got invalid query: %v\n", query)
+		return ""
+	}
+	payloadBody, _ := json.Marshal(map[string]string{
+		"query": query,
+	})
+	payload := bytes.NewBuffer(payloadBody)
+
+	client := &http.Client{Timeout: 1 * time.Minute}
+	var vaultUrl = os.Getenv("VAULT_URL")
+	var url = vaultUrl + vaultId + "/query"
+
+	request, _ := http.NewRequest("POST", url, payload)
+	request.Header.Add("Accept", "application/json")
+	request.Header.Add("Content-Type", "application/json")
+	request.Header.Add("Authorization", "Bearer "+token)
+
+	response, err := client.Do(request)
+	if err != nil {
+		log.Printf("Got error on Skyflow Validation request: %v\n", err.Error())
+		return ""
+	}
+
+	if response.StatusCode != http.StatusOK {
+		responseBody, _ := io.ReadAll(response.Body)
+		defer response.Body.Close()
+		log.Printf("Got status on Skyflow Validation: %v\n", response.StatusCode)
```

Comment:
Can we change message to `Unable/Fail to call Skyflow API status code : <>  and message :  <>`